ketcher-autotests/tests/Structure-Creating-&-Editing/SMARTS-attributes/Bond-attributes/bond-attributes.spec.ts

```diff
@@ -224,4 +242,181 @@ test.describe('Checking bond attributes in SMARTS format', () => {
       '[#6](-[#6])(-[#6])~,=[#6]',
     );
   });
+
+  test('Setting custom query - directional bond "up or unspecified"', async ({
+    page,
+  }) => {
+    /**
+     * Test case: https://github.com/epam/Indigo/issues/1281
+     * Description: "or unspecified"("?") quantifier should be supported
+     */
+    await setAndCheckBondProperties(
+      page,
+      setCustomQueryForBond,
+      '/?',
+      '[#6](-[#6])(-[#6])/?[#6]',
+    );
+  });
+});
+
+test.describe('Checking converting bond attributes to custom query', () => {
+  test.beforeEach(async ({ page }) => {
+    const numberOfBond = 2;
+    await waitForPageInit(page);
+    await drawStructure(page);
+    await page.keyboard.press('Escape');
+    await doubleClickOnBond(page, BondType.SINGLE, numberOfBond);
+    await waitForBondPropsModal(page);
```

Comment:
Maybe we could exract a common function for this beforeEach to reduce replication since it is used more than once